.eslintrc.js

```diff
@@ -10,11 +15,32 @@ module.exports = {
 	globals: {
 		asyncRequire: true
 	},
+	plugins: [
+		'import'
+	],
+	settings: {
+		'import/resolver': {
+			[ customResolverPath ]: {}
+		}
+	},
 	rules: {
 		camelcase: 0, // REST API objects include underscores
+		// NOTE: Some import rules are errors in client (Webpack's resolution) and
+		//       warnings in server (Node's module resolution).
```

Comment:
Can't we still define these all as errors and expect the server-specific configuration to override them?